### PR TITLE
Updating splunk version

### PR DESCRIPTION
**Pull request by nwang92**
This branch will fail until the bits have been published, but I'm updating the version and hash anyways because we'll want it to tie off the release.

### CODE REVIEW

**Comment by nwang92** [comment]
Hmm maybe this PR should go directly into `master`?

**Review by arctan5x — DISMISSED** [review]

**Review by hendolim — DISMISSED** [review]

**Review by hendolim — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -409,7 +409,7 @@ CUR_SIZE=$$(docker image inspect $1:latest --format='{{.Size}}') ; \
 EDGE_SIZE=$$(docker image inspect splunk/splunk:edge --format='{{.Size}}') ; \
 echo "current $1 image size = "$$CUR_SIZE ; \
 echo "edge image size = "$$EDGE_SIZE ; \
-if [[ $$CUR_SIZE -gt $$EDGE_SIZE*120/100 ]] ; then echo "current image size is 20% more than edge image" ; exit 1 ; fi
+if [[ $$CUR_SIZE -gt $$EDGE_SIZE*140/100 ]] ; then echo "current image size is 40% more than edge image" ; exit 1 ; fi
```
Comment (hendolim) [inline Makefile:412]: Is it the splunk binary that has gotten so big?

Comment (arctan5x) [reply Makefile:412]: it was ad hoc change in order to accommodate the release ;)

Comment (arctan5x) [reply Makefile:412]: Also, Splunk 8.0.0 vs 7.3.2 saw about 30% increase for linux distro when unextraced.

**Review by arctan5x — APPROVED** [review]

**Pull request merged**